tutorials/13-real-time.ipynb

```diff
@@ -459,6 +465,31 @@
     "plt.colorbar()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Like we mentioned all the way back in notebook 01, it is possible to view these weights using more sophisticated tools such as niwidgets"
```

Comment:
Great!

Reply:
@CameronTEllis we should also update the requirements file with this new package.

Reply:
@CameronTEllis please also create a PR on NeuroLibre (https://github.com/neurolibre/brainiak-tutorials) with notebooks 4 and  5 to resolve the memory error.